app/controllers/publishers/vacancies/documents_controller.rb

```diff
@@ -68,7 +68,10 @@ def uploading_more_documents?
 
   def after_document_delete_path
     if vacancy.supporting_documents.none?
-      new_organisation_job_document_path(vacancy.id, back_to_review: params[:back_to_review], back_to_show: params[:back_to_show])
+      organisation_job_build_path(vacancy.id,
+                                  :include_additional_documents,
+                                  back_to_review: params[:back_to_review],
+                                  back_to_show: params[:back_to_show])
```

Comment:
Unrelated to the bug:
We are changing the redirection path when the last document is deleted from the vacancy documents list. Our BA proposed this during review so we quickly changed it.